python/pyspark/ml/classification.py

```diff
@@ -183,7 +191,7 @@ def getThresholds(self):
         If :py:attr:`thresholds` is set, return its value.
         Otherwise, if :py:attr:`threshold` is set, return the equivalent thresholds for binary
         classification: (1-threshold, threshold).
-        If neither are set, throw an error.
```

Comment:
We should probably harmonize with the Scala side and return the default value yes?